src/components/BackToHomeButton.jsx

```diff
@@ -0,0 +1,18 @@
+import Link from "next/link";
+
+import { LuArrowLeft } from "react-icons/lu";
+
+const BackToHomeButton = () => {
+  return (
+    <Link href="/">
```

Comment:
link should be included with the div. you can add classname to the Link component